pkg/executor/join/hash_join_v2.go

```diff
@@ -1047,6 +1047,27 @@ func (e *HashJoinV2Exec) reset() {
 	}
 }
 
+func (e *HashJoinV2Exec) collectSpillStats() {
+	if e.stats == nil || !e.spillHelper.isSpillTriggered() {
+		return
+	}
+
+	round := e.spillHelper.round
+	for len(e.stats.spill.totalSpillBytesPerRound) < round+1 {
```

Comment:
It looks a little strange for using "for loop" here, since in the following code, only the last round info are updated.

Reply:
> It looks a little strange for using "for loop" here, since in the following code, only the last round info are updated.

okk, I will replace it with `if`